code/modules/mob/living/carbon/monkey/punpun.dm

```diff
@@ -0,0 +1,65 @@
+/mob/living/carbon/monkey/punpun //except for a few special persistence features, pun pun is just a normal monkey
+	name = "Pun Pun" //C A N O N
+	unique_name = 0
+	var/ancestor_name
+	var/ancestor_chain = 1
+	var/relic_hat	//Note: these two are paths
+	var/relic_mask
+	var/memory_saved = 0
+	var/list/pet_monkey_names = list("Pun Pun", "Bubbles", "Mojo", "Forever")
+
```

Comment:
@KorPhaeron SS13 is full of -all- pop culture type references, soo, eat a dick? :3